core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -1215,7 +1216,15 @@ object AutoTuner extends Logging {
       val reader = new BufferedReader(new InputStreamReader(fsIs))
       val fileContent = Stream.continually(reader.readLine()).takeWhile(_ != null).mkString("\n")
       loadClusterPropertiesFromContent(fileContent)
-    } finally {
+    }
+    catch {
+      // In case of missing file for cluster properties, default properties are used.
+      // Hence, catching and logging as a warning
+      case _: FileNotFoundException =>
```

Comment:
I'm assuming there is a chance something else bad happens here, maybe a misformatted file or issue  getting the file FileSystem.get returns IOException for instance. its probably safer to catch nonFatal errors here and log and return None

Reply:
Right! in this method we can capture `IOException` which is more generic.
The caller method catches the NoFatal(e) to cover other sort of exceptions that we are not aware of

Reply:
Thanks @tgravescs @amahussein . This has been corrected